src/imitation/policies/interactive.py

```diff
@@ -44,34 +66,35 @@ def _choose_action(self, obs: np.ndarray) -> np.ndarray:
         return np.array([self.action_key_to_index[key]])
 
     def _get_input_key(self) -> str:
+        """Obtains input key for action selection."""
         print(
             "Please select an action. Possible choices in [ACTION_NAME:KEY] format:",
-            ", ".join(
-                [f"{n}:{k}" for n, k in zip(self.action_names, self.action_keys)]
-            ),
+            ", ".join([f"{n}:{k}" for k, n in self.action_keys_names.items()]),
         )
 
         key = input("Your choice (enter key):")
-        while key not in self.action_keys:
+        while key not in self.action_keys_names.keys():
```

Comment:
similar to above (OK to leave as-is)
```suggestion
        while key not in self.action_keys_names:
```